tests/Oriflame.PolicyBuilder.Xml.Tests/Definitions/Common/ForwardRequestsTests.cs

```diff
@@ -8,22 +8,12 @@ namespace Oriflame.PolicyBuilder.Xml.Tests.Definitions.Common
     public class ForwardRequestsTests
     {
         [Theory]
-        [InlineData(5)]
-        public void CreatesCorrectPolicy(int timeoutSeconds)
+        [InlineData("5")]
+        public void CreatesCorrectPolicy(string timeoutSeconds)
```

Comment:
Parameter and `InlineData` attribute not needed anymore.

Reply:
You are right, removed